### ISSUE TITLE
deploying easyocr in a serverless: How to avoid the Read-only trap ?

### ISSUE THREAD

**Issue by AvekIA** [JaidedAI/EasyOCR#605]
The way `easyocr.Reader` behave make it write into files. that's not allowed in a serverless space.
I managed to deploy and make a local server works by following [this tutorial](https://medium.com/geekculture/deploying-pytorch-model-as-a-serverless-service-339b4b93e517).

Everything succeed, even the `serverless deploy` ! but when I try to use POST method with POSTMAN I receive a forbidden access and when I test the API Gateway, I get the error bellow.

So in order to work, `easyocr.Reader` seems to need to write something on the `model` directory (I moved it locally). It works on local, it works on localhost, it doesn't work on a docker image on AWS. I need to find a way to avoid writing something on the files after the image is build, is it possible ?

```
{
  "errorMessage": "[Errno 30] Read-only file system: 'model/temp.zip'",
  "errorType": "OSError",
  "stackTrace": [
    "  File \"/var/task/handler.py\", line 25, in predict\n    reader = easyocr.Reader(['en'], model_storage_directory=\"model\",\n",
    "  File \"/var/task/easyocr/easyocr.py\", line 90, in __init__\n    download_and_unzip(detection_models[detector_model]['url'], detection_models[detector_model]['filename'], self.model_storage_directory, verbose)\n",
    "  File \"/var/task/easyocr/utils.py\", line 586, in download_and_unzip\n    urlretrieve(url, zip_path, reporthook=reporthook)\n",
    "  File \"/var/lang/lib/python3.8/urllib/request.py\", line 257, in urlretrieve\n    tfp = open(filename, 'wb')\n"
```

**Comment by amroghoneim** [comment]
I am having the same issue.

I am currently downloading the model files into a models directory manually, then I specify 
model_storage_directory = 'models/', download_enabled = False
when intializing Reader.
I then inlclude this folder in the Dockerfile while building the image for it to work.

But pushing such large models to git(considering CI/CD) is giving me a headache, so I want the models to download automatically when running the container.

**Comment by mansi2606** [comment]
Hi, @amroghoneim @AvekIA  can you guys tell me how much time is it taking to fetch a response using this method?

**Comment by AvekIA** [comment]
@amroghoneim, I did the same, I also registered craft_mlt_25k.pth and it did the job for me.
@mansi2606 takes me 5-7 secondes to get a response from my lambda. (I send the name of the image to the lambda and it pull out the image from an s3 to analyze it and send me an answer so 5-7 don't even count the time to put and delete the image on the s3, it's kinda slow yup but the results are satisfaying so...). :)
